inspector/src/components/actionTabs/tabs/propertyGrids/animations/editorControls.tsx

```diff
@@ -85,10 +117,14 @@ export class EditorControls extends React.Component<
         });
     };
 
+    // Recount animations

```

Comment:
Inconsistent commenting. comments? Seems like you're going back and forth between /* */ and // not sure why.

Reply:
The single-line comments were aimed just as context comments. But I update those to match the /**/ format. np.